src/main/scala/wdl4s/wdl/expression/WdlStandardLibraryFunctions.scala

```diff
@@ -77,8 +76,8 @@ trait WdlStandardLibraryFunctions extends WdlFunctions[WdlValue] {
   def basename(params: Seq[Try[WdlValue]]): Try[WdlString] = {
 
     val arguments: Try[(WdlValue, WdlValue)] = params.toList match {
-      case Success(f) :: Nil => Success(f, WdlString(""))
-      case Success(f) :: Success(s) :: Nil => Success(f, s)
+      case Success(f) :: Nil => Success((f, WdlString("")))
```

Comment:
If I remember what eta expansion is, no; this was running afoul of relying on auto-tupling.